airflow/models.py

```diff
@@ -3024,6 +3024,25 @@ def get_active_runs(self, session=None):
         return active_dates
 
     @provide_session
+    def get_num_active_runs(self, external_trigger=None, session=None):
```

Comment:
Are you sure your count() function is faster: http://docs.sqlalchemy.org/en/latest/orm/query.html and https://stackoverflow.com/questions/14754994/why-is-sqlalchemy-count-much-slower-than-the-raw-query ?

As performance indeed isn't a real issue. How many active dag runs will you find? 200? 400? 1000? I'm not particularly convinced it warrants the extra complexity of adding this function or not using DagRun.find.